org.civicrm.reqangular/src/common/decorators/angular-date/datepicker-popup.js

```diff
@@ -16,6 +16,23 @@ define([], function () {
                  */
                 attrs.datepickerPopup = HR_settings.DATE_FORMAT;
 
+                scope.$watch('isOpen', function (value) {
+                    if (value) {
+                        scope.popupPosition = $position.position(element);
+                        scope.popupPosition.top += scope.popupPosition.height;
+
+                        var popupWidth = 316;
```

Comment:
Two things:
1) This seems like a magic constant. Do we need this exact number, or can we extrapolate it in some way? If we cannot, please move it to a "constant" with a comment as to why we need this number
2) Please always declare `var`s at the beginning of the function